js/src/hooks/useAdsCurrency.js

```diff
@@ -37,15 +36,15 @@ import useGoogleAdsAccount from './useGoogleAdsAccount';
  *
  * @see useStoreCurrency
  *
- * @return {{adsCurrencyConfig: Object, hasFinishedResolution: boolean | undefined, formatAmount: Function}} The currency object.
+ * @return {{adsCurrencyConfig: Object, formatAmount: Function}} The currency object.
  */
 export default function useAdsCurrency() {
 	const storeCurrencySetting = useStoreCurrency();
-	const { googleAdsAccount, hasFinishedResolution } = useGoogleAdsAccount();
+	const { googleAdsAccount } = useGoogleAdsAccount();
 
 	// Apply store's foramtting config with the Ad's currency and symbol.
```

Comment:
💅

typo - foramtting -> formatting

Reply:
Thanks! Fixed in https://github.com/woocommerce/google-listings-and-ads/pull/1255/commits/5d66532b073a4be2b890481ba06f32e5735f26a6.